codespan/src/span.rs

```diff
@@ -137,6 +137,15 @@ where
     }
 }
 
+impl<I> From<Span> for Range<I>
```

Comment:
Dang - this seems valid on nightly at least! 🤔

Reply:
The error makes sense to me? `impl From Span for Range<I>` means that it can be turned into a range over _any_ type, including types not in std. Try `impl From Span for Range<usize>` instead.

Reply:
I dunno, I think it should work, but the rule might have been loosened recently 🤔

Reply:
I'll just do `Range<usize>` though